RELEASING.md

```diff
@@ -93,116 +69,48 @@ green, you may create a release as follows:
 
-1. [Create a pull request from your
-   fork](https://help.github.com/articles/creating-a-pull-request-from-a-fork/)
-   using the branch containing your commit(s). Assign the pull request to
-   yourself for merging. Request the appropriate reviewers.
+1. Review the PR, apply the appropriate label(s), and request a review from googleapis/yoshi-ruby.
 
-1. After your pull request has passed all checks and been approved by reviewers,
-   **Squash and merge** it. This will trigger a build job on [Circle
-   CI](https://circleci.com/gh/googleapis/google-cloud-ruby).
-
-1. Wait until the [Circle CI
-   build](https://circleci.com/gh/googleapis/google-cloud-ruby) has
-   passed for the squashed pull request commit in master.
-
-1. On the [google-cloud-ruby releases
-   page](https://github.com/googleapis/google-cloud-ruby/releases),
-   click [Draft a new
-   release](https://github.com/googleapis/google-cloud-ruby/releases/new).
-   Complete the form as follows for each gem in your pull request. You should
-   refer to the GitHub view of your squashed commit for content.
-
-   1. In the `Tag version` field, enter the tag name in the following format:
-
-       ```
-       <gem>/v<version>
-       ```
+1. Repeat steps 3 through 12 if you are releasing multiple gems.
 
-   1. In the `Target` dropdown, select the squashed commit from your pull
-      request.
-
-   1. In the `Release title` field, enter `<gem> <version>`, e.g.
-      `google-cloud-bigquery 1.2.0`.
-
-   1. In the description text area, paste in the bullet-point list from the
-      `CHANGELOG.md` for the gem and release.
-
-1. Click `Publish release`. This will trigger a build job on [Circle
-   CI](https://circleci.com/gh/googleapis/google-cloud-ruby) for the
-   tag.
-
-1. Wait until the [Circle CI
-   build](https://circleci.com/gh/googleapis/google-cloud-ruby) has
-   passed for the tag.
+1. If you updated `google-cloud/google-cloud.gemspec` for a version change to
+   any gem, repeat steps 4 through 12 for the `google-cloud` gem.
 
-1. Confirm that the new version is displayed on the [google-cloud-ruby gh-pages
-   doc
-   site](https://http://googleapis.github.io/google-cloud-ruby/docs/). Or, to
-   save time when releasing many gems at once, check out the gh-pages branch and
-   pull repeatedly to confirm that it has received the new docs.
+1. If you updated `stackdriver/stackdriver.gemspec` for a version change to any
+   gem, repeat steps 4 through 12 for the `stackdriver` gem.
 
-   If the gh-pages branch has not been updated, inspect the Kokoro build logs to
-   confirm that the rake `post` task succeeded.
+1. After your pull request has passed all checks and been approved by reviewers,
+   **Squash and merge** it. This will trigger a build job on [Kokoro](#Checking-the-status-of-kokoro-builds), which will create the [release](https://github.com/googleapis/google-cloud-ruby/releases) and build and push the gem to [rubygems](https://rubygems.org/).
```

Comment:
There is nothing in this stepset preventing pull requests from being merged simultaneously. That isn't a problem is it? Kokoro can handle the merging of multiple release PRs in parallel, correct?

Reply:
It shouldn't be a problem. If the number of builds queued in parallel exceeds the number of VM's reserved, it will just queue the remaining builds.